envpool/sokoban/astar_log.cc

```diff
@@ -75,37 +81,49 @@ void RunAStar(const std::string& level_file_name,
         assert(action >= 0 && action < 4);
         loglinestream << action;
         steps++;
+        int curr_x = node->player_x;
+        int curr_y = node->player_y;
+        int delta_x = node->kDelta.at(action).at(0);
+        int delta_y = node->kDelta.at(action).at(1);
+        if (curr_x != prev_x + delta_x || curr_y != prev_y + delta_y) {
+          correct_solution = false;
+        }
+        prev_x = curr_x;
+        prev_y = curr_y;
+      }
+      if (!correct_solution) {
+        loglinestream << "INCORRECT_SOLUTION_FOUND,search_steps" << std::endl;
```

Comment:
Do you mean to output the string `search_steps` ?

Reply:
There's also no comma separating action list from INCORRECT_SOLUTION_FOUND.

Reply:
resolved, thanks!